tests/test_mathematical.py

```diff
@@ -442,6 +443,49 @@ def test_sign_boolean():
         dpnp.sign(dpnp_a)
 
 
+@pytest.mark.parametrize(
+    "data",
+    [[2, 0, -2], [1.1, -1.1]],
+    ids=["[2, 0, -2]", "[1.1, -1.1]"],
+)
+@pytest.mark.parametrize("dtype", get_all_dtypes(no_complex=True))
+@pytest.mark.usefixtures("allow_fall_back_on_numpy")
+def test_signbit(data, dtype):
```

Comment:
please update `tests/test_sycl_queue.py::test_1in_1out` and `tests/test_usm_type.py::test_1in_1out` with new functions

Reply:
`signbit` is added. however, `proj` cannot be added since NumPy does not have it.

Reply:
Can it be added as new test in `test_sycl_queue.py` and in `test_usm_type.py`

Reply:
I meant a new separate test for `dpnp.proj` there.